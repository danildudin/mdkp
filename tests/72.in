18 6
307 577 355 323 802 344 594 386 750 726 183 254 435 46 781 45 334 529 
988 901 426 598 10 428 146 403 323 693 23 769 68 893 276 915 919 355 2432
288 882 114 558 815 876 546 735 311 847 254 568 297 669 504 471 300 341 5824
173 375 989 634 184 402 508 248 775 165 171 636 20 984 943 429 381 329 2748
468 568 77 575 429 197 131 228 613 400 152 688 704 679 200 729 417 758 5663
279 182 184 925 397 471 482 255 39 856 109 62 234 76 863 54 151 286 5795
981 798 974 878 755 411 574 128 535 371 458 301 732 755 895 757 933 709 6304
