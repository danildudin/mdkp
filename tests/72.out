4566
8
4 5 6 7 8 9 12 17 
